auth.py

```diff
@@ -1,13 +1,15 @@
 import bcrypt
 import os
-from flask_login import login_user, UserMixin
+from flask_login import login_user, UserMixin, current_user
 
 USER_NAME = os.getenv("USER_NAME")
 SALTED_PASSWORD = os.getenv("SALTED_PASSWORD")
 CREDENTIAL_STORE = {
     USER_NAME: SALTED_PASSWORD
 }
 
+AUTH_ENABLED = os.getenv("AUTH_ENABLED", "True").lower() == "true"
```

Comment:
Consider reusing the way `PUBLIC_ONLY` is defined: https://github.com/weilu/rpf-country-dash/blob/main/queries.py#L8